src/applications/personalization/profile/components/ProfileSubNav.jsx

```diff
@@ -43,6 +44,12 @@ const ProfileSubNav = ({ isInMVI, isLOA3, routes, clickHandler = null }) => {
     }
     const { href } = e.detail;
     history.push(href);
+    if (showPaperlessDelivery && mobileNavRef?.current) {
+      const button = mobileNavRef.current.shadowRoot
+        ?.querySelector('va-accordion > va-accordion-item')
+        ?.shadowRoot?.querySelector('button');
+      button?.click();
```

Comment:
I think this is a nice improvement to the component and I'm not sure if the design system team considered it when it was being build. I'm going to chat with them about if we should make this default behavior in mobile view.

In the mean time, I want to suggest future proofing your solution in case the component does end up doing this internally. I think if it did, adding a click here might actually open it back up. 

Is it possible to programmatically update the `open` prop on `va-accordion-item` to false instead of using `.click()`?